src/python/tests/core/bot/tasks/corpus_pruning_task_test.py

```diff
@@ -499,6 +498,36 @@ def test_prune(self):
         'gs://{}/corpus/libFuzzer/test_fuzzer/'.format(
             self.backup_bucket) + '%s.zip' % today)
 
+  def get_mock_record_compare(self, project_qualified_name, method, sources,
+                              tags, initial_corpus_size, corpus_size,
+                              initial_edge_coverage, edge_coverage,
+                              initial_feature_coverage, feature_coverage):
+    """Helper function. Given all of the expected stats, returns a function
+    that will compare them to an instance of CorpusPruningStats."""
+
+    def compare(stats):
+      return (stats.project_qualified_name == project_qualified_name and
+              stats.method == method and stats.sources == sources and
+              stats.tags == tags and
+              stats.initial_corpus_size == initial_corpus_size and
+              stats.corpus_size == corpus_size and
+              stats.initial_edge_coverage == initial_edge_coverage and
+              stats.edge_coverage == edge_coverage and
+              stats.initial_feature_coverage == initial_feature_coverage and
+              stats.feature_coverage == feature_coverage)
+
+    return compare
+
+  def test_untrusted_cross_pollination_stats(self):
+    """Test that the untrusted runner gets correct stats to BigQuery"""
+    self.mock.record_cross_pollination_stats.side_effect = (
+        self.get_mock_record_compare('test_fuzzer', 'random', 'test2_fuzzer',
```

Comment:
I don't think this is testing anything as you might expect? .side_effect = Func allows you to mock out a function with another implementation, but in this case you're doing some comparisons which have no effect. Did you mean to make them self.assertEqual() checks instead rather than plain == comparison?

Also, we can just stick the pollination test in the main test_prune in the interests of time as this integration test can take a while to run.

Reply:
Yea, good catch. I meant to do self.assertEqual() :).